packages/medusa/src/services/line-item.js

```diff
@@ -30,6 +31,9 @@ class LineItemService extends BaseService {
     /** @private @const {ProductVariantService} */
     this.productVariantService_ = productVariantService
 
+    /** @private @const {PricingService} */
+    this.pricingService_ = pricingService
```

Comment:
if you rebase develop into your branch, this file has been migrated to TS. So you might get some conflict :)